addons/mail/models/res_users.py

```diff
@@ -116,7 +116,7 @@ def _unsubscribe_from_channels(self):
             public channels are mailing list (e-mail based) and so users should always receive
             updates from public channels until they manually un-subscribe themselves.
         """
-        self.mapped('partner_id.channel_ids').filtered(lambda c: c.public != 'public').write({
+        self.mapped('partner_id.channel_ids').filtered(lambda c: c.public != 'public' and c.channel_type != 'chat').write({
```

Comment:
Instead of `!= 'chat'` I would do `== 'channel'`, as channel was the target of the original commit here.
This has the benefit to also include livechat by default where we probably want the same change to be done for consistency.

Reply:
done. you may check now.

Reply:
changes are done.

Reply:
Thanks, this should cover it well. Just make sure to adapt the comments too as I posted on the diff.